src/ImmutableX.ts

```diff
@@ -111,18 +111,26 @@ export class ImmutableX {
   }
 
   public updateCollection(
-    requestParameters: API.CollectionsApiUpdateCollectionRequest,
+    signer: EthSigner,
+    address: string,
```

Comment:
CollectionsApiUpdateCollectionRequest.address had intellisense comment specifying this is a collection contract address. Not sure if we need to be more specific about this address.

Reply:
definitely! `address` is ambiguous